crawler.py

```diff
@@ -39,11 +39,11 @@
 CHROME = 'chrome'
 FIREFOX = 'firefox'
 
-MAJESTIC_URL = "http://downloads.majesticseo.com/majestic_million.csv"
-WEEK_IN_SECONDS = 604800
 RESTART_RETRIES = 5
 
-OUR_PYFUNCEBLE_CONFIG = {"share_logs": False}
+TRANCO = Tranco(cache=True, cache_dir='.tranco')
```

Comment:
Look at where we stored cache files for PyFunceble. Should probably do the same for Tranco. Something like `os.path.join(out_path,'.tranco')`?

Reply:
Good idea, changed this and added to next commit